docs/source/quick-start/opf.rst

```diff
@@ -34,7 +34,7 @@ using the :class:`.ModelFactory`.
 
 .. literalinclude:: ../../examples/opf/create-model-example.py
 
-The resulting ``model`` will be an instance of :class:`.CLAModel`.
+The resulting ``model`` will be an instance of :class:`.SPTMModel`.
```

Comment:
SPTM is not too informative. Should we consider calling it StreamingModel or PredictionModel?